Dorime

Silverexperience opened the pull request:
## What Does This PR Do
Este PR añade las nuevas armas del chapalin que solicitarion en issues.

**El Unholy bleassing**: Lo mismo que el dark blessing, el brazo de changeling, pero con la textura del tentaculo del otro brazo del changeling.
**El Egyptian staff**:Un palo egipcio que sirve para golpear a tus contrincantes con un daño moderado y ser una reskin del null rod.
**El  Pride-struck hammer**: Un martillo con daño leve pero al lanzarlo tiene un daño mayor, como el fedora pero con otro sprite y para el cinturon.

## Why It's Good For The Game
Mas variedad de cosas para el capellan, para que esta gente pueda inventarse aun mas religiones y no veamos la tipicas religiones sangrientas que se copian de Nar'sie.

## Images of changes
### `Palo egipcio`
![imagen](https://user-images.githubusercontent.com/58746682/78171088-385e4b00-7454-11ea-9ce1-f08b2d5b8cdb.png)
### `Martillo endemoniado`
![imagen](https://user-images.githubusercontent.com/58746682/78171148-575cdd00-7454-11ea-9db1-9d431df4a8ff.png)
### `Brazo tentaculo del capellan`
![imagen](https://user-images.githubusercontent.com/58746682/78171183-6479cc00-7454-11ea-8820-c010c2a46362.png)
### `Nuevos objetos en el null rod del capellan`
![imagen](https://user-images.githubusercontent.com/58746682/78171664-15806680-7455-11ea-8d14-015973711be4.png)
![imagen](https://user-images.githubusercontent.com/58746682/78171736-35178f00-7455-11ea-921a-21db2ce68244.png)

## Changelog
:cl:
add: Añade nuevos objetos al capellan.
imageadd: Añade nuevos sprites para objetos del capellan.
/:cl:

Silverexperience committed:
Dorime

Silverexperience commented:
#515

SagaStars commented:
Wait this means... DOBLE HOLORPARASITO??!?!?!?

Silverexperience commented:
> 
> 
> Wait this means... DOBLE HOLORPARASITO??!?!?!?

Se me olvido pensar eso, pero creo que no se puede.
y si se puede lo quito.

Silverexperience commented:
Revisando el codigo si no me equivoco esto:
```
for(var/mob/living/simple_animal/hostile/guardian/G in GLOB.living_mob_list)
		if(G.summoner == user)
			to_chat(user, "You already have a [mob_name]!")
			return
```
deberia hacer que no puedas tener dos, asique no deberia exisitir la posiblidad de doble holoparasito.

SagaStars commented:
> deberia hacer que no puedas tener dos, asique no deberia exisitir la posiblidad de doble holoparasito.

Awwww

Tilk1 commented:
Habia leido tambien que propusieron agregar la Ouija como seleccionable de la nul rod

Interesante la idea del holo parasito pero no estoy seguro que implicancias tendra en cuestiones de balance... Capaz convenga hacer un PR independiente para ello

Silverexperience commented:
> 
> 
> Habia leido tambien que propusieron agregar la Ouija como seleccionable de la nul rod
> 
> Interesante la idea del holo parasito pero no estoy seguro que implicancias tendra en cuestiones de balance... Capaz convenga hacer un PR independiente para ello

El holoparasito es mas un escudo que nada, y no puede curar mucho porque te mata el cerebro en unos 5 o 10 clicks y encima no puede curar a su invocador, no lo veo muy buffado la verdad, y la bilbia hace el mismo efecto de curar.
Tampoco puede romper objetos, vidrios o paredes.

SagaStars commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,148 @@
+			C.adjustToxLoss(-5)
+			C.adjustBrainLoss(15)
+			heal_cooldown = world.time + 20
+			if(C == summoner)
```
No deberia estar el If con el return arriba? De otra manera no serviria de nada, ya que primero le curas y luego te dice que no puedes.

Silverexperience replied:
Esto me pasa por copiar todo del guardian curador.
Ya lo cambio.

R31L3 replied:
Puede el Holyparasite actual dañar/destruir runas de cultistas?

Silverexperience replied:
No, se me olvido, pero lo voy a añadir.

Silverexperience committed:
Ameno

SagaStars commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,148 @@
+	sharp = 0
+	force = 10
+
+/obj/item/nullrod/egyptian
```
Ya que esta es un staff seria mejor que fuera como el monk's staffs, para que asi sea tenga sus buenas estadisticas y funcione mas como una skin. Por que ya sabes, la mitad de holy weapons del chaplain son re-skins

SagaStars replied:
`/obj/item/nullrod/claymore/bostaff/egyptian`

Silverexperience replied:
Asi fue como lo pidio el del issue, un null rod con otra skin.

SagaStars commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,148 @@
+	w_class = WEIGHT_CLASS_HUGE
+	attack_verb = list("bashes", "smacks", "whacks")
+
+/obj/item/nullrod/hammmer/pride_hammer
```
`slot_flags = SLOT_BELT` Por el bien de la patria.

Silverexperience replied:
Añadido.

SagaStars commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,148 @@
+	desc = "A tentacle made by ancient dark gods."
+	hitsound = 'sound/effects/splat.ogg'
+	sharp = 0
+	force = 10
```
Hm no lo se 10 es bastante mediocre, y mas con que se te quede pegado en el brazo yo diría que lo dejes con el mismo daño que la armblade, plus como le quitastes el sharp ya seria mucho y quedaría olvidadisimo, yeah dejale el mismo daño de la armblade.

Silverexperience replied:
Lo dejo igual que el armblade.

ghost commented:
El mensaje de "The card are still unused" debería ser "The card is still unused" o "The cards are still unused" si son más de una

SagaStars commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,148 @@
+	item_state = "pride"
+	lefthand_file = 'icons/hispania/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/hispania/mob/inhands/weapons_righthand.dmi'
+	force = 9
```
COMO QUE 9, SILVER VEN ACA PARA UNAS NALGADAS. Dejale el daño que ya tiene la nullrod (15) osea no lo toques, ya que el chaplain tiene un martillo el cual hace 15, por que elegiría este?

Silverexperience replied:
Este al lanzarlo hace mas daño, esa es la gracia de este martillo.

SagaStars replied:
hMHmhmHMm, bien bien. Subele un poquito el daño como 22 y que en la desc del arma te mencione algo al respecto de que haga mas daño.

Silverexperience replied:
Hara lo mismo que un fedora.

ghost commented:
También quiero agregar que el sprite de la carta no parece una carta del tarot parece más una carta de CaH

Silverexperience commented:
> 
> 
> También quiero agregar que el sprite de la carta no parece una carta del tarot parece más una carta de CaH

Ese era el sprite del codigo que estoy porteando esto, pero si tienes alguna sugerencia de aspecto mejor puedo cambiarlo.

Silverexperience committed:
Abajo el cristianismo, Arriba el protestantismo

Silverexperience committed:
Merge branch 'Branch' of https://github.com/Silverexperience/Paradise into Branch

SagaStars commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,150 @@
+				C.adjustFireLoss(-5, robotic=1)
+				C.adjustOxyLoss(-5)
+				C.adjustToxLoss(-5)
+				C.adjustBrainLoss(15)
```
BROOOOOOOOOOOOOoOoOoo XDD, QUE TE MATA EL CEREBRO DE 5 O 7 GOLPES

Silverexperience replied:
Sere yo muy nerfeador, pero me parece bien asi.

SagaStars replied:
DON'T 
![image](https://user-images.githubusercontent.com/55407528/78178666-020dd500-742e-11ea-93f4-56a9dc9ecf8f.png)

Silverexperience replied:
Bien, se deja en 5 y solo a los bendecidos.

Silverexperience committed:
Cambios a cosas de este PR que hago porque si no no funciona.pdf

Silverexperience committed:
Vengo corriendo con este commit para desearos feliz jueves.

Silverexperience committed:
Otro PR para el pozo.

Silverexperience committed:
Bye bye holyparasite

Silverexperience committed:
No te quiero aqui.

Silverexperience committed:
Bye bye ocean.

Silverexperience commented:
Listo el WIP y la division del PR.

Silverexperience committed:
Merge remote-tracking branch 'upstream/master' into Branch

Silverexperience committed:
Revert "Bye bye ocean."

This reverts commit 345d2fd227615595d9b37c3be19b21bcb35ef068.

Silverexperience committed:
Revert "Revert "Bye bye ocean.""

This reverts commit cce6f405ab5db37bb0b8411f70d48b2b255d543e.

Spircen commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,31 @@
+
+/obj/item/nullrod/egyptian
+	name = "egyptian staff"
+	desc = "A tutorial in mummification is carved into the staff. You could probably craft the wraps if you had some cloth."
```
Esta descripción me hace pensar que existe una interacción entre el staff y una sheet de cloth. ¿La puedes cambiarla? Solo lo ultimo.

Spircen replied:
Otra cosa, de favor que se pueda equipar en la espalda como el resto de staff que ya existen.

Silverexperience replied:
Ahora puede ponerse en la espalda, aunque los sprites no se si son muy bonitos, tambien quite la parte de la descripcion.

Spircen commented on code/HISPANIA/game/objects/items/weapons/holy_weapons.dm:
```diff
@@ -0,0 +1,31 @@
+	lefthand_file = 'icons/hispania/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/hispania/mob/inhands/weapons_righthand.dmi'
+	force = 3
+	throwforce = 25
```
Es bastante alto el valor pero tiene sentido, si la gente lo recibe evita el daño por completo, me imagino es mas para atacar mobs que personas.

Spircen reviewed (commented):
Fuera de esos dos cambios todo parece en orden para mergearlo.

Silverexperience committed:
Las arañas son aracnidos y canbios al Staf

Quita parte de la descripcion del staff egipcio y lo permite poner en la espalda.

Silverexperience committed:
*Cambios *Staff

Silverexperience committed:
Merge remote-tracking branch 'upstream/master' into Branch

Silverexperience committed:
No conflictos

Silverexperience committed:
Merge pull request #8 from Helixis/master

Actualizacion
